docs/source/_toc.yml

```diff
@@ -21,6 +21,7 @@ chapters:
       - file: tutorials/InvertLensEquation
       - file: tutorials/Parameters
       - file: tutorials/Simulators
+      - file: tutorials/Microlensing
```

Comment:
The TOC still has the filename wrong so it's not rendering into the docs

```suggestion
      - file: tutorials/microlensing
```

Reply:
There was a weird issue where the capitalization wasn't tracked (it was capitalized correctly on my Macbook)
![image](https://github.com/user-attachments/assets/add4b194-9fc6-4925-9512-6b28c71e81af)

As a quick work around, did the following: rename the file, commit, rename it back (with proper capitalization) and commit again. It looks like that fixed the issue!